storage/s3-sdk/requirements-test.txt

```diff
@@ -1,3 +1,3 @@
 backoff==1.10.0
 pytest==6.2.4
-google-cloud-storage==1.38.0
+google-cloud-storage==1.40.0
```

Comment:
Put this in a separate PR? Why is renovate not making the update here?

Reply:
Done. Renovate bot updates the latest release in #6197. Once this test fix is checked in, #6197 should be unblocked.